tests/apps/pubsub-subscriber/app.go

```diff
@@ -120,6 +135,7 @@ func subscribeHandler(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusInternalServerError)
 		json.NewEncoder(w).Encode(appResponse{
 			Message: err.Error(),
+			Status:  "DROP",
```

Comment:
`DROP`ed messages are error messages but not retried.
https://github.com/dapr/dapr/blob/master/pkg/runtime/runtime.go#L1109